multiformats/js-multihash
# feat: add flow type annotations

Also configure build/test/lint scripts for Flow typed JS

 - Run flow check after linting
 - Use AEgir config to use babel-loader for webpack builds
 - Use different babel config for browser builds to avoid inlining source maps

This PR supersedes #47. I didn't want to hijack @gozala's PR. This PR is just a cleaned up version of his work. All credits go to him.

@gozala please review if it is still doing what it's supposed to do. I've changed the npm scripts a bit to run the build before the test runs. I also removed the hack around Mocha, it's now part of AEgir in the `flow` branch.
